release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -125,6 +125,15 @@ With this release, OVN-Kubernetes uses a new `DNSNameResolver` custom resource t
 
 For more information, see xref:../networking/openshift_network_security/configuring-egress-firewall-ovn.adoc#nw-coredns-egress-firewall[Improved DNS resolution and resolving wildcard domain names].
 
+[id="ocp-4-16-q-in-q-support"]
+==== Supporting double-tagged packets (QinQ) 
+
+This release introduces 802.1q-in-802.1q also known as QinQ support. QinQ introduces a second VLAN tag, where the service provider designates the outer tag for their use, offering them flexibility, while the inner tag remains dedicated to the customer’s VLAN. When two VLAN tags are present in a packet, the outer VLAN tag can be either 802.1q or 802.1ad. The inner VLAN tag must always be 802.1q.
+
+{product-title} supports both the standardized (802.1ad/802.1q) and non standardized (802.1q/802.1q) way of implementing double tagging.  
```

Comment:
we need to mention which cards support 802.1ad/802.1q and which ones only 802.1q/802.1q